deepspeed/moe/sharded_moe.py

```diff
@@ -474,13 +474,17 @@ def __init__(self,
         self.timers = SynchronizedWallClockTimer()
         self.wall_clock_breakdown = False
 
-        self.use_tutel = use_tutel and TUTEL_INSTALLED
+        self.use_tutel = use_tutel and TUTEL_INSTALLED and gate.k == 1
 
         if self.use_tutel:
             logger.info('Using Tutel optimizations.')
         elif use_tutel and not TUTEL_INSTALLED:
             logger.warning("Tutel optimization requested but not installed. "
                            "Proceeding without Tutel.")
+        elif use_tutel and TUTEL_INSTALLED and gate.k != 1:
+            logger.warning(
```

Comment:
Can we wrap this in a `if torch.distributed.get_rank() ==0:`?

Reply:
Yeah it is possible. But I wonder should we also wrap other warnings and infos? For example, L480 and L482-483?